migrations/util/index.js

```diff
@@ -0,0 +1,88 @@
+const Umzug = require('umzug')
+const Sequelize = require('sequelize')
+const mysql = require('mysql2/promise')
+
+const models = require('../../models')
+
+/**
+ * Executes all pending migrations.
+ * @return {Promise} A promise that resolves when all migrations are complete
+ */
+module.exports.performMigrations = async sequelize => {
```

Comment:
Where/when does this get called? Does it not need to be explicitly called in server.js?

Reply:
Currently, just in the below function - only used for verification right now. Though it might make sense to have this be automatic instead of requiring `$> sequelize db:migrate` in a terminal. @wadefagen given that you're doing deploys, thoughts?

Reply:
Is there a downside to having it be automatic?
If someone doing a deploy that includes changed schema and doesn't run migrations, could they see unexpected behavior?

Reply:
There probably shouldn't be any major downsides. Yep, they'd see unexpected behavior eventually. My only thought is that we might want to do migrations as part of a standardized deploy process that includes snapshotting the database in case something goes wrong, but even then, if they're applied before the app starts, they should all have been applied already. So... Probably safe to automate this.